test/tests/device/isSafari.js

```diff
@@ -3,14 +3,6 @@
 import { isSafari } from '../../../src/device';
 
 describe('isSafari', () => {
-    it('should return true when userAgent contains Safari and isChrome function returns false', () => {
```

Comment:
Removed since some non-safari user agents strings could contain the `safari` word:
Reference here https://developers.whatismybrowser.com/useragents/explore/software_name/samsung-browser/